The-SMIRNOFF-force-field-format.md

```diff
@@ -182,6 +182,8 @@ Currently, only classical fixed point charge models are supported, but future ex
 
 ### `<LibraryCharges>`: Library charges for polymeric residues and special solvent models
 
+.. warning:: This functionality is not implemented in the current toolkit.
```

Comment:
Suggest "This functionality is not yet implemented and will appear in a future version of the toolkit."

Reply:
Hm, what if we move `<LibraryCharges>` and `<ChargeIngrementModel>` to a "proposed extensions to SMIRNOFF" section?

Reply:
> Suggest "This functionality is not yet implemented and will appear in a future version of the toolkit."

That sounds a lot better. Changed in 56ad21f 

> Hm, what if we move <LibraryCharges> and <ChargeIngrementModel> to a "proposed extensions to SMIRNOFF" section

I think the layout of the page right now does a really good job of grouping these sections in a way that gives a big-picture view of what we seek to accomplish, even if it's not all ready right now.

Reply:
OK, but let's keep this in mind if there is confusion due to the mixed structure!